pywincffi/kernel32/handle.py

```diff
@@ -120,3 +120,50 @@ def WaitForSingleObject(hHandle, dwMilliseconds):
     error_check("WaitForSingleObject")
 
     return result
+
+
+def GetHandleInformation(hObject):
+    """
+    Returns properties of an object handle.
+
+    .. seealso::
+
+        https://msdn.microsoft.com/en-us/library/windows/desktop/ms724329(v=vs.85).aspx
+
+    :param handle hObject:
+        A handle to an object whose information is to be retrieved.
+    """
+    input_check("hObject", hObject, Enums.HANDLE)
+    ffi, library = dist.load()
+
+    lpdwFlags = ffi.new('LPDWORD')
```

Comment:
I'd prefer we remain consistent with regards to `'` vs `"`.  I've tried to document the reasoning behind this here: https://github.com/opalmer/pywincffi/commit/f9946a7224351fce178bde550f8b23b7ff5be7d9#diff-dc4147bbe08af630a0a380407d3097c2R7